public/data/pools-fluxes-examples-limited.csv

```diff
@@ -1,37 +1,37 @@
+flux,Precipitation over land ,Precipitation over land ,111000,99000,120000,precipitation_over_land,cubic kilometers per year,landPrecipitation.png,100+ kostenlose Regen-Duschen und Regen-Bilder (pixabay.com),"Water released from clouds in the form of rain, freezing rain, sleet, snow, or hail that falls over the land surface.",https://pixabay.com/es/photos/lluvia-ducha-campo-agricultura-5112946/,this is a test
+flux,Precipitation over ocean,Precipitation over ocean,380500,320000,460000,precipitation_over_ocean,cubic kilometers per year,oceanPrecipitation.png,Water Waves Horizon - Pixabay free pictures,"Water released from clouds in the form of rain, freezing rain, sleet, snow, or hail that falls over the ocean.",https://pixabay.com/pt/photos/agua-ondas-horizonte-oceano-1867234/,this is a test
+flux,Ocean evaporation,Ocean evaporation,420500,350000,510000,ocean_evaporation,cubic kilometers per year,oceanEvaporation.png,niKo SnOOze | flickr.com,The conversion of ocean water to water vapor at the interface between the ocean surface and the atmosphere.,https://www.flickr.com/photos/le_k/26640707858/in/photolist-GA9Aih-NYbFfc-LfCNDG-pzm74Z-Cs1Ltd-Jh2bab-27QPBGc-2eGNq2Z-28uzkfF-ZiuW2p-2bemoTi-af1eBU-2dYaXQi-28Uoz6L-CZJGwf-7svMub-pqu1WY-Nwx3Pe-GcnwJV-tQpubj-2acCw7d-fLj2H7-kuQUeB-2ekUoPx-bTaFuF-ktfXjj-coRzNu-u1jL3D-22kJKwS-21vAvTf-pZ3ZKG-NPeVyL-AH1HBR-ySLw6v-9kmYGP-ku68a6-EyXSb9-cGMMaq-7sxFVQ-26HWK4j-puMHtd-jyvM7x-22v6P5u-EjaCxP-HGRnoy-2bemoyk-mzZ8gA-ZYFACE-neAc7Y-dxVZQt,this is a test
+flux,Ocean circulation,Ocean circulation,7100000,6100000,9100000,ocean_circulation,cubic kilometers per year,oceanCirculation.png,NASA Goddard Space Flight Center | Dynamic Earth - Ocean Currents,The movement and mixing of water horizontally between ocean basins and vertically between mixed and deep zones.,https://www.flickr.com/photos/24662369@N07/7402445224,this is a test
+example,Great Salt Lake,Great Salt Lake,18.9,,,great_salt_lake,cubic kilometers,greatSaltLake.png,"Hannah E. McIlwain | Utah Water Science Center, USGS",,https://www.usgs.gov/media/images/great-salt-lake-may-reach-historic-low-soon,this is a test
+example,Dead Sea,Dead Sea,147,,,dead_sea,cubic kilometers,deadSea.png,https://www.usgs.gov/media/images/kafrein-reservoir-captures-runoff-near-dead-sea-jordan,,https://www.usgs.gov/media/images/kafrein-reservoir-captures-runoff-near-dead-sea-jordan,this is a test
+example,Lake Winnepeg,Lake Winnepeg,284,,,lake_winnepeg,cubic kilometers,lakeWinnepeg.png,"Jacques Descloitres, MODIS Rapid Response Team, NASA/GSFC",,https://visibleearth.nasa.gov/images/72159/lake-winnipeg-manitoba-canada,this is a test
+example,Lake Erie,Lake Erie,480,,,lake_erie,cubic kilometers,lakeErie.png,The Great Lakes: Lake Erie | EPA.gov,,https://www.epa.gov/greatlakes/lake-erie,this is a test

```

Comment:
Noticing we have no text for the examples.  Do we want to give some context here?

Reply:
Hmm we could add a bit about where they are located? I think that could be in a separate PR.